### ISSUE TITLE
[TW-1848] verbose=context token

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1863]
_David Patrick on 2016-08-28T22:54:56Z says:_ 

 I find that if context is set, and I forget about it, that the resulting task lists can be .. confusing. I would use a verbose=context setting, but as yet, unless I am mistaken, none such thing exists.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2016-08-28T22:54:56Z
Modified: 2017-10-12T03:00:58Z
```

**Comment by taskwarrior** [comment]
_Tomas Babej on 2017-02-08T19:44:48Z says:_ 

 Not sure if this deserves its own verbose option, but indeed something like:

```
$ task list
...
Tasks restricted to context 'family'.
```

printed for any command that uses_context would be nice.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2017-02-09T13:41:04Z says:_ 

 Tomas, I agree.

**Comment by taskwarrior** [comment]
_David Patrick on 2017-02-10T05:02:37Z says:_ 

 I agree too, sort of, but what if you *didn't* want that (usually a good idea) "Tasks restricted to context 'xyz'". How would use control the display of that (valuable) information? verbosity tokens come to mind.

**Comment by mrossinek** [comment]
I am happy to look into this! Just to make sure, before I start:
a) Shall I implement this **without** using a new verbosity token?
b) or shall I go ahead and add this as a new valid token in `Context::verbose`?
If you tend to (b): would it be fine to parse this in a footnote?

**Comment by tbabej** [comment]
@mrossinek Hey Max, do you still want to tackle this? :slightly_smiling_face:
(b) is the way to go here - we should use a new verbosity token which, if active, dumps the information about the current context as a footnote

**Comment by tbabej** [comment]
Shoot, looks like everybody involved in this conversation missed the fact this was possible since 2015 (implemented in 3a77a5f291ca21a4111a0549a9431fae3256c2bf).

Keeping this open to ensure this is documented.

**Comment by mrossinek** [comment]
Hey @tbabej! Thanks for the ping. I admit I did not notice that this was already implemented. I don't recall exactly what I was looking at 2 years ago though..
I see you also just opened a PR to address the rest of this issue, thanks! :slightly_smiling_face:
